packages/nitro-protocol/deployment/deploy.ts

```diff
@@ -0,0 +1,80 @@
+import {GanacheDeployer, ETHERLIME_ACCOUNTS} from '@statechannels/devtools';
+import {Wallet} from 'ethers';
+
+import {writeGasConsumption} from '../test/test-helpers';
+import countingAppArtifact from '../artifacts/contracts/CountingApp.sol/CountingApp.json';
+import erc20AssetHolderArtifact from '../artifacts/contracts/test/TestErc20AssetHolder.sol/TestErc20AssetHolder.json';
+import ethAssetHolderArtifact from '../artifacts/contracts/test/TestEthAssetHolder.sol/TestEthAssetHolder.json';
+import nitroAdjudicatorArtifact from '../artifacts/contracts/NitroAdjudicator.sol/NitroAdjudicator.json';
+import singleAssetPaymentsArtifact from '../artifacts/contracts/examples/SingleAssetPayments.sol/SingleAssetPayments.json';
+import testAssetHolderArtifact1 from '../artifacts/contracts/test/TESTAssetHolder.sol/TESTAssetHolder.json';
+import testAssetHolderArtifact2 from '../artifacts/contracts/test/TESTAssetHolder2.sol/TESTAssetHolder2.json';
+import testForceMoveArtifact from '../artifacts/contracts/test/TESTForceMove.sol/TESTForceMove.json';
+import testNitroAdjudicatorArtifact from '../artifacts/contracts/test/TESTNitroAdjudicator.sol/TESTNitroAdjudicator.json';
+import tokenArtifact from '../artifacts/contracts/Token.sol/Token.json';
+import trivialAppArtifact from '../artifacts/contracts/TrivialApp.sol/TrivialApp.json';
+
+const deploy = async () => {
+  const deployer = new GanacheDeployer(Number(process.env.GANACHE_PORT));
+
+  const nitroAdjudicatorDeploymentGas = await deployer.etherlimeDeployer.estimateGas(
+    nitroAdjudicatorArtifact as any
+  );
+  writeGasConsumption('NitroAdjudicator.gas.md', 'deployment', nitroAdjudicatorDeploymentGas);
+  console.log(
+    `\nDeploying NitroAdjudicator... (cost estimated to be ${nitroAdjudicatorDeploymentGas})\n`
+  );
```

Comment:
The diff isn't easy to read because of the extension change. These are the new lines.

Reply:
A good practice here is to separate the two changes into separate commits: one changes the extension but no logic, the second changes the logic.

Reply:
As you wish #3083